sdk/storage/storage-queue/src/QueueServiceClient.ts

```diff
@@ -111,6 +114,24 @@ export interface ServiceListQueuesSegmentOptions {
  * @class QueueServiceClient
  */
 export class QueueServiceClient extends StorageClient {
+  /**
+   * Creates an instance of QueueServiceClient.
+   *
+   * @param {string} connectionString Connection string for an Azure storage account.
+   * @param {NewPipelineOptions} [options] Optional. Options to configure the HTTP pipeline.
+   * @memberof QueueServiceClient
+   */
+  public static fromConnectionString(connectionString: string, options?: NewPipelineOptions) {
```

Comment:
Should define return type `: QueueServiceClient` for this method definition

Reply:
Fixed.